email_mailbox/src/reducers/__tests__/labels.js

```diff
@@ -118,4 +118,12 @@ describe('Label actions - UPDATE', () => {
     const newState = labelReducer(state, action);
     expect(newState).toBe(new Map({}));
   });
+
+  it('should remove group of label', () => {
+    const state = initState(myLabels);
+    const labelIds = ['1', '6'];
+    const action = actions.removeLabels(labelIds);
+    const newState = labelReducer(state, action);
+    expect(newState.size).toBe(5);
+  });
```

Comment:
Add test:
- should not remove labels nonexisten ... (check the result)
- should not remove labels by wrong emailIds ... (use labelsIds as number array: [1,6])

Reply:
Could you explain this one a bit more `should not remove labels by wrong emailIds`?